src/main/scala/org/broadinstitute/dsde/firecloud/service/DataUseRestrictionSupport.scala

```diff
@@ -1,18 +1,25 @@
 package org.broadinstitute.dsde.firecloud.service
 
 import com.typesafe.scalalogging.slf4j.LazyLogging
+import org.broadinstitute.dsde.firecloud.model.DataUse.DiseaseOntologyNodeId
 import org.broadinstitute.dsde.rawls.model.WorkspaceJsonSupport.AttributeNameFormat
 import org.broadinstitute.dsde.rawls.model._
 import spray.json.DefaultJsonProtocol._
 import spray.json._
 
+import scala.util.{Failure, Success, Try}
+
 trait DataUseRestrictionSupport extends LazyLogging {
 
   private val booleanCodes: Seq[String] = Seq("GRU", "HMB", "NCU", "NPU", "NDMS", "NAGR", "NCTRL", "RS-PD")
   private val listCodes: Seq[String] = Seq("DS", "RS-POP")
-  val durFieldNames: Seq[String] = booleanCodes ++ listCodes ++ Seq("RS-G")
+  private val genderCodes: Seq[String] = Seq("RS-G", "RS-FM", "RS-M")
+  val durFieldNames: Seq[String] = booleanCodes ++ listCodes ++ genderCodes
 
+  // TODO: This value will change depending on what is implemented for GAWB-2758
+  val diseaseLabelsAttributeName: AttributeName = AttributeName.withLibraryNS("DS-labels")
```

Comment:
This will need to change depending on what the field is actually named in GAWB-2758